src/lib/badges/Badge.svelte

```diff
@@ -1,26 +1,24 @@
 <script lang="ts">
   import classNames from 'classnames';
-  import CloseButton from '../utils/CloseButton.svelte';
```

Comment:
This CloseButton component's style is hard to modify, so I directly make changes on a <button> element (see beow).